components/src/core/DrawerLayout/DrawerLayout.tsx

```diff
@@ -59,7 +65,10 @@ const DrawerLayoutRender: React.ForwardRefRenderFunction<unknown, DrawerLayoutPr
         >
             <div ref={ref} className={clsx(defaultClasses.root, classes.root)} {...otherDivProps}>
                 <div className={clsx(defaultClasses.drawer, classes.drawer)}>{drawer}</div>
-                <div className={clsx(defaultClasses.content, classes.content)} style={style}>
+                <div
+                    className={clsx(defaultClasses.content, classes.content, padding === 0 && defaultClasses.expanded)}
```

Comment:
for conditional classes in clsx, we usually prefer the object syntax:
```suggestion
                    className={clsx(defaultClasses.content, classes.content, {[defaultClasses.expanded]: padding === 0})}
```

Reply:
applies to the one above too